types/properties-reader/index.d.ts

```diff
@@ -18,6 +18,9 @@ declare namespace PropertiesReader {
         each<T>(iterator: (this: T, key: string, value: Value) => void, scope: T): Reader;
         getAllProperties(): { [key: string]: Value; };
         clone(): Reader;
+        save(destFile: string): Promise<any>;
```

Comment:
According to the documentation and source, there's also a callback that can be provided as a second parameter.

Reply:
Yes, you are right. Fixed